src/docs/releases/1.6.0.md

```diff
@@ -39,7 +43,7 @@ If your are using setup recipes and want to enable `Features`, be sure to explic
 ### New `OrchardCore.Notifications` Module
 New feature was added to provide a way to send notification messages to the users. [Click here](../reference/modules/Notifications/README.md) for more info about notifications.
 
-### `OrchardCore.Contents` Feature
+### `OrchardCore.Contents` Module
```

Comment:
The term `Feature` is right

Yes it is also a module but whose main feature is `OC.Features`

Reply:
I would say `OrchardCore.Contents` is a module `Contents` is the feature.

Reply:
Okay no problem if Module is better in this context, was just to point out that maybe Feature was okay as the feature Id is also `OrchardCore.Contents`.

Just find another typo `OrchardCore.Roles feature in enabled` => `is enabled`.

Did you try my suggestion about using `SearchIndexViewModel : ShapeViewModel` in place of the shape factory?

Reply:
@jtkech yes I changed it and it works. Just a note that `SearchIndexViewModel` is a parameter in the action method, if it okay to inherit from `ShapeViewModel` since we allow binding. I think I want to introduce `SearchViewModel` for binding. thoughts?

Reply:
As you want if compatible with what we had before

Reply:
@jtkech please check out the updated code. I removed the `ShapeFactory` which simplify the controller

Reply:
Looks good! Okay if it still works

Reply:
Yes it works. Please approve if you have no additional changes